docs/guides/configure-error-suppression.mdx

```diff
@@ -8,12 +8,8 @@ description: How to configure runtime compilation in Qiskit Runtime.
 
 Runtime compilation techniques optimize and transform your circuit to minimize errors. Runtime compilation adds some classical pre-processing overhead to your overall runtime. Therefore, it is important to achieve a balance between perfecting your results and ensuring that your job completes in a reasonable amount of time.
```

Comment:
It's weird to talk about optimization when it's only done in V1, which is deprecated. But presumably that'd be fixed in the options PR.

Reply:
I think we should fix that in this PR, then the changes will be reflected in Options.